src/client/api/types.ts

```diff
@@ -19,12 +19,14 @@ export interface ILanguageServer extends Disposable {
 
 export const IPythonApiProvider = Symbol('IPythonApi');
 export interface IPythonApiProvider {
+    onDidActivatePythonExtension: Event<void>;
```

Comment:
Not fully sure if I'm suggesting this. But does this seem to you like it should be on IPythonExtensionChecker? That has everything relevant to checking if the extension is installed or active.

Reply:
No can do, its the PythonApiProvider that activates the Python extension. So only PythonAPIProvider knows if the python extension was activated or not.
VS Code doesn't fire events when extensions load.
Here we're only interested in situations when we load the python extension & other parts of the code can then react to that.